postings/models.py

```diff
@@ -61,6 +65,37 @@ def save_image(self, image):
         """
         pass
 
+    #Takes in a company name and bucket location for upload
+    #and uploads the file to S3, and returns the s3 url
+    def load_logo(self, company, bucket):
```

Comment:
Add docstring in following format describing what the function does. Here's an example
```
    """Fetches rows from a Bigtable.

    Retrieves rows pertaining to the given keys from the Table instance
    represented by big_table.  Silly things may happen if
    other_silly_variable is not None.

    Args:
        big_table (dict): An open Bigtable Table instance.
        keys (list[str]): A sequence of strings representing the key of each table row
            to fetch.
        other_silly_variable: Another optional variable, that has a much
            longer name than the other args, and which does nothing.

    Returns:
        A dict mapping keys to the corresponding table row data
        fetched. Each row is represented as a tuple of strings. For
        example:

        {'Serak': ('Rigel VII', 'Preparer'),
         'Zim': ('Irk', 'Invader'),
         'Lrrr': ('Omicron Persei 8', 'Emperor')}

        If a key from the keys argument is missing from the dictionary,
        then that row was not found in the table.

    Raises:
        IOError: An error occurred accessing the bigtable.Table object.
    """
```

Reply:
shouldn't take on the parameter bucket. it should be hardcoded